mappings/net/minecraft/test/StructureTestUtil.mapping

```diff
@@ -38,3 +44,14 @@ CLASS net/minecraft/class_4525 net/minecraft/test/StructureTestUtil
 		ARG 0 pos
 		ARG 1 radius
 		ARG 2 world
+	METHOD method_22368 (ILnet/minecraft/class_2338;Lnet/minecraft/class_3218;)V
+		ARG 0 altitude
+		ARG 1 pos
+		ARG 2 world
+	METHOD method_22369 createStructure (Ljava/lang/String;Lnet/minecraft/class_3218;)Lnet/minecraft/class_3499;
+		ARG 0 structureId
+		ARG 1 world
+	METHOD method_23646 createArea (Lnet/minecraft/class_2338;Lnet/minecraft/class_2338;I)Lnet/minecraft/class_3341;
+		ARG 0 startPos
+		ARG 1 endPos
+		ARG 2 horizontalSize
```

Comment:
What does "area" mean here? What's the "horizontal size" and why isn't it determined by start and end positions?

Reply:
The area is a `BlockBox` used for positioning structures. I've revised the parameters a bit, but basically the values are:

`pos` = Pos1 of the box. This is the `pos` parameter passed down from other methods.
`size` = Width/Height/Depth of the box (passed down as the size of the structure)
`margin` = Extra margin to expend the area by along the X+Z axis.

The final box's dimensions are calculated as follows:
`min = pos - [margin, 3, margin]
max = pos + size + [margin - 1, 30, margin - 1]
`